packages/aws-cdk-lib/aws-ec2/lib/vpc-endpoint-service.ts

```diff
@@ -95,6 +101,7 @@ export class VpcEndpointService extends Resource implements IVpcEndpointService
 
     this.vpcEndpointServiceLoadBalancers = props.vpcEndpointServiceLoadBalancers;
     this.acceptanceRequired = props.acceptanceRequired ?? true;
+    this.contributorInsightsEnabled = props.contributorInsightsEnabled ?? true;
```

Comment:
```suggestion
    this.contributorInsightsEnabled = props.contributorInsights;
```

If `props.contributorInsights` is undefined, we should pass it to CfnVPCEndpointService prop as undefined.